Dockerfile.arm

```diff
@@ -21,7 +25,9 @@ RUN find . -type d -name obj | xargs -r rm -r
 RUN dotnet publish Jellyfin.Server --configuration Release --output="/jellyfin" --self-contained --runtime linux-arm "-p:GenerateDocumentationFile=false;DebugSymbols=false;DebugType=none"
 
 
-FROM debian:buster-slim
+FROM multiarch/qemu-user-static:x86_64-arm as qemu
+FROM arm32v7/debian:buster-slim
+COPY --from=qemu /usr/bin/qemu-arm-static /usr/bin
 RUN apt-get update \
  && apt-get install --no-install-recommends --no-install-suggests -y ffmpeg \
  libssl-dev ca-certificates \
```

Comment:
```suggestion
 && apt-get install --no-install-recommends --no-install-suggests -y jellyfin-ffmpeg
	libssl1.0.0 \
	ca-certificates \
	libfontconfig1 \
	libfreetype6 \
	libomxil-bellagio0 \
	libomxil-bellagio-bin \
	libraspberrypi0 \
```

Reply:
This is what I'm building now, I may have to adjust packages after netcore finishes, I haven't run these packages against debian yet. I still had to add --network-timeout 100000 after I fixed my... issues.